Library/ViewModels/PledgeCTAContainerViewViewModel.swift

```diff
@@ -130,19 +130,18 @@ public final class PledgeCTAContainerViewViewModel: PledgeCTAContainerViewViewMo
 // MARK: - Functions
 
 private func pledgeCTA(project: Project, backing: Backing?) -> PledgeStateCTAType {
-  let uuid = deviceIdentifier(uuid: UUID())
-  let optimizelyVariant = AppEnvironment.current.optimizelyClient?
-    .variant(
-      for: OptimizelyExperiment.Key.pledgeCTACopy,
-      userId: uuid,
-      isAdmin: AppEnvironment.current.currentUser?.isAdmin ?? false
-    )
-
   guard let projectBacking = backing, project.personalization.isBacking == .some(true) else {
     if currentUserIsCreator(of: project) {
       return PledgeStateCTAType.viewYourRewards
     }
 
+    let optimizelyVariant = AppEnvironment.current.optimizelyClient?
```

Comment:
Moved the activation of the experiment to _after_ the `guard` that checks whether the project has already been backed or not - this prevents us from activating the experiment when we know it won't be shown.